.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: memo-backend
+        working-directory: memo
```

Comment:
```suggestion
```

Ich glaube das ist gar nicht mehr noetig. So wie es aktuell hier steht, versucht er gradle in einem `memo` Unterordner auszufuehren.